### PR TITLE
feature: panels/keywords listing

### PR DESCRIPTION
**Pull request by aconanlai**

### CODE REVIEW

File: src/redux/modules/keywords.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+export const receiveKeywords = (keywords) => {
+  return {
+    type: 'RECEIVE_KEYWORDS',
+    keywords,
+  };
+};
+
+export const fetchKeywords = () => {
```
Comment (jamaljeantobias) [inline src/redux/modules/keywords.js]: NICE!

**Comment by jamaljeantobias** [comment]
Looking GOOD!

File: src/components/Episode/EpisodeContainer.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const NotOnPageYet = () => (
 
 const getSlugFromPath = (path) => {
   const parts = path.split('/');
+  console.log(parts);
```
Comment (jamaljeantobias) [inline src/components/Episode/EpisodeContainer.js]: Remove the unnecessary  console.log

Comment (aconanlai) [reply src/components/Episode/EpisodeContainer.js]: done!

**Pull request merged**